.github/workflows/linux.yml

```diff
@@ -20,11 +20,12 @@ jobs:
       image: openquantumsafe/ci-ubuntu-jammy:latest
     env:
       MAKE_PARAMS: "-j 18"
+      LIBOQS_BRANCH="0.10.1"
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
       - name: Full build
-        run: OQSPROV_CMAKE_PARAMS=${{ matrix.cmake-params}} ./scripts/fullbuild.sh
+        run: OQSPROV_CMAKE_PARAMS=${{ matrix.cmake-params}} ../scripts/fullbuild.sh
```

Comment:
wow, did you just fix a bug here?

Reply:
No -- it actually looks like I introduced one. Thanks for catching this! What I don't understand at all is why/how this is not getting reflected in github CI: Does anyone see whether the linux tests actually ran??? I don't seem to "find" them either in the test list above, nor within the Actions tab...

Reply:
It looks to me like that the tests didn't run on [36038b8](https://github.com/open-quantum-safe/oqs-provider/pull/429/commits/36038b8447ce1e7c42a47f64e72d1a2254d36cb8) (and hence this PR) because the  commit message ended in "[skip ci]". It looks like they ran on the previous commit, [cad0016](https://github.com/open-quantum-safe/oqs-provider/pull/429/commits/cad0016a7d6eef7bae504ad9e88904353e1024bb) and all passed.

Reply:
Thanks for the pointer @SWilson4! Now I also see (should find in the future) such screw-ups.

Reply:
@thb-sb So the above now has triggered a CI error. But thanks for the heads-up anyway. And Thanks to @SWilson4 for pointing out that the github UI is not highlighting all CI errors....

Reply:
Marking as resolved now that the irregularity has been ironed out.